docs/reference/transform/checkpoints.asciidoc

```diff
@@ -22,21 +23,25 @@ indices. This check is done based on the interval defined in the transform's
 +
 If the source indices remain unchanged or if a checkpoint is already in progress
 then it waits for the next timer.
++
+If changes are found a checkpoint is created.
 
 . Identifies which entities have changed.
```

Comment:
in my PR I changed "entities" to "entities and/or time buckets". Alternatively we could generalize and say "Identifies which data has changed" or "Identifies which documents need an update"

Reply:
@hendrikmuhs Thanks for spotting! I've added your original change (entities and/or time buckets) as I think it is the better solution because it highlights time buckets explicitly.